Solutions/MimecastTTP/SolutionMetadata.json

```diff
@@ -0,0 +1,20 @@
+{
+    "publisherId": "azuresentinel",

```

Comment:
The tier is mentioned as Partner. So the publisherID needs to be changed according to the publisherID of the partner.